### PR TITLE
Optimise `monitor` icons/add [`monitor-`]`play`/`dot`/`check` icons

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Review by karsa-mistmere — APPROVED** [review]
Nice ones, they all look good to me!

Maybe we could also add some others?
Edit: pushed `monitor-x`, `monitor-pause` and `monitor-stop`.

File: icons/monitor-play.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m10 7 5 3-5 3V7z" />
```
Comment (jguddas) [inline icons/monitor-play.svg:12]:
```suggestion
  <path d="m10 7 5 3-5 3Z" />
```

**Comment by karsa-mistmere** [comment]
> > Edit: pushed `monitor-x`
> 
> @karsa-mistmere Is the `x` slightly too big? Relative to other `*-x` icons https://lucide.dev/icons?search=x

Yeah, I'm afraid our Xs aren't very consistent in size:
image

But you're right, better make this one the same as in file, folder etc.

**Comment by danielbayley** [comment]
> Yeah, I'm afraid our Xs aren't very consistent in size

I noticed. Same as the question marks, I think fixing all of those are for separate PRs…

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]
Nice work @danielbayley!!

**Pull request merged**